test/file01.tsx

```diff
@@ -0,0 +1,38 @@
+import { INPUT_TYPES } from '@xsite-ui/prequal-funnels.constants';
+import React, { useState } from 'react';
+
+import { AddressQuestionProps } from '../types';
+import FullAddress from './components/full-address';
+import ManualAddress from './components/manual-address';
+import { Container, ToggleButton, ToggleContainer } from './styles';
+import { getToggleCtaTextToRender, getToggleTextToRender } from './utils';
+
+const Address = ({ onBlur, questions, stepAnswers }: AddressQuestionProps) => {
```

Comment:
The 'onBlur', 'questions', and 'stepAnswers' props are used but not defined in the exported service.